addons/im_livechat/models/discuss_channel.py

```diff
@@ -671,3 +671,40 @@ def _action_unfollow(self, partner=None, guest=None, post_leave_message=True):
             # sudo: discuss.channel - last operator left the conversation, state must be updated.
             channel_sudo.livechat_end_dt = fields.Datetime.now()
             Store(bus_channel=self).add(channel_sudo, "livechat_end_dt").bus_send()
+
+    def _forward_human_agent(self, agent, create_member_params=None, inviting_partner=None):
+        """Forward the conversation, initially handled by a bot, to a human agent.
```

Comment:
I don't think it's relevant whether the conversation was previously handled by a bot or not.